app/src/main/java/org/mozilla/fenix/session/NotificationSessionObserver.kt

```diff
@@ -5,38 +5,39 @@
 package org.mozilla.fenix.session
 
 import android.content.Context
-import mozilla.components.browser.session.Session
-import mozilla.components.browser.session.SessionManager
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.map
+import mozilla.components.browser.state.selector.privateTabs
+import mozilla.components.lib.state.ext.flowScoped
+import mozilla.components.support.ktx.kotlinx.coroutines.flow.ifChanged
 import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.sessionsOfType
 
 /**
  * This observer starts and stops the service to show a notification
  * indicating that a private tab is open.
  */
-
+@ExperimentalCoroutinesApi
```

Comment:
Not sure if you've considered this already: we have components that use flows and to avoid the annotation from being littered through the codebase, we add [this suppressor](https://github.com/mozilla-mobile/android-components/blob/master/components/feature/syncedtabs/build.gradle#L30-L34) to the `build.gradle`.

Reply:
fenix's gradle file is a lot more intimidating :P I'll try to find a spot to add the suppressor tomorrow

Reply:
Do you know what might be causing this error? 
```
Class kotlinx.coroutines.ExperimentalCoroutinesApi is not an experimental API marker annotation
```

Reply:
@NotWoods hm yeah, it seems this is not needed anymore?
```
tasks.withType(org.jetbrains.kotlin.gradle.tasks.AbstractKotlinCompile).all {
    kotlinOptions.freeCompilerArgs += ["-Xuse-experimental=kotlinx.coroutines.ExperimentalCoroutinesApi"]
}
```

Reply:
For now I'll switch back to adding the annotations manually.

Reply:
👍